packages/fxa-auth-server/test/local/routes/subscriptions/stripe.js

```diff
@@ -211,13 +238,18 @@ describe('subscriptions stripeRoutes', () => {
       support: {
         ticketPayloadLimit: 131072,
       },
+      contentful: MOCK_CONTENTFUL,
```

Comment:
Actually, is this still necessary given that you're completely mocking the call to capability manager below?

Reply:
It's not necessary, I'll remove them - thanks!